.buildkite/release-builds.yml

```diff
@@ -19,3 +19,6 @@ steps:
     plugins: *common_plugins
     notify:
     - slack: "#build-and-ship"
+    retry:
+      manual:
+        allowed: false
```

Comment:
Same as the other PR I commented on this, I'd suggest adding a `# comment` on those saying that we disable job retry because even if that job failed we'd recommend to re-trigger a new build from the ReleaseV2 UI in that case rather than retrying that particular individual job from Buildkite UI.